src/main/java/li/cil/scannable/common/scanning/FluidBlockScannerModule.java

```diff
@@ -34,48 +32,37 @@ public int getEnergyCost(final ItemStack module) {
         return CommonConfig.energyCostModuleFluid;
     }
 
-    @OnlyIn(Dist.CLIENT)
+    @Environment(EnvType.CLIENT)
     @Override
     public ScanResultProvider getResultProvider() {
         return ScanResultProviders.BLOCKS.get();
     }
 
-    @OnlyIn(Dist.CLIENT)
+    @Environment(EnvType.CLIENT)
     @Override
     public float adjustLocalRange(final float range) {
         return range * Constants.BLOCK_MODULE_RADIUS_MULTIPLIER;
     }
 
-    @OnlyIn(Dist.CLIENT)
+    @Environment(EnvType.CLIENT)
     @Override
     public Predicate<BlockState> getFilter(final ItemStack module) {
         validateFilter();
         return filter;
     }
 
-    @OnlyIn(Dist.CLIENT)
+    @Environment(EnvType.CLIENT)
     private void validateFilter() {
         if (filter != null) {
             return;
         }
 
         final List<Predicate<BlockState>> filters = new ArrayList<>();
-        final ITagManager<Fluid> tags = ForgeRegistries.FLUIDS.tags();
-        if (tags != null) {
-            tags.getTagNames().forEach(tag -> {
-                if (!CommonConfig.ignoredFluidTags.contains(tag.location())) {
-                    filters.add(new FluidTagScanFilter(tag));
-                }
-            });
-        }
+        Registry.FLUID.getTagNames().forEach(namedTag -> {
+            if (!CommonConfig.ignoredFluidTags.contains(namedTag.location())) {
+                filters.add(new FluidTagScanFilter(namedTag));
+            }
+        });
         filter = new BlockCacheScanFilter(filters);
     }
-
-    @OnlyIn(Dist.CLIENT)
-    @SubscribeEvent
-    public static void onModConfigEvent(final ModConfigEvent configEvent) {
```

Comment:
Since the cache invalidation was ported for the common ores one, any reason it's not also done here?

Reply:
It was an oversight in both cases.